src/Chapter1.hs

```diff
@@ -614,8 +626,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = mod (last2Digit (abs n)) 10 + div (last2Digit (abs n)) 10
+  where
+    last2Digit :: Int -> Int
+    last2Digit l = mod l 100
```

Comment:
That is a wonderful solution! Great job 🤩 

You could also could find `divMod` useful. `divMod` is the standard function that combines both `div` and `mod`. And as you do both on `10` it could be helpful!